tests/test_tarball_generation.py

```diff
@@ -1,21 +1,39 @@
 #!/usr/bin/env python3
 
 import os
-import subprocess
-import time
+import re
 import requests
-import signal
-import tarfile
 import shutil
+import signal
+import subprocess
 import sys
+import tarfile
+import time
+
 
 # 1. Extract the tarball
-tarball_path = "target/unitycatalog-0.1.0.tar.gz"
+tarball_path = "target/unitycatalog-{version}.tar.gz"
```

Comment:
will this work with version being `x.y.z-snapshot`?

Reply:
@tlm365

Reply:
I think it should.

Reply:
@tdas as long as it's in quotes (in `version.sbt`) -> version := "**this_will_be_captured**"